ShellPkg/Application/Shell/ShellParametersProtocol.c

```diff
@@ -1182,7 +1183,13 @@ UpdateStdInStdOutStdErr (
 
         if (!ErrUnicode && !EFI_ERROR (Status)) {
           TempHandle = CreateFileInterfaceFile (TempHandle, FALSE);
-          ASSERT (TempHandle != NULL);
+          // MU_CHANGE [BEGIN] - CodeQL change
+          if (TempHandle == NULL) {
+            ASSERT (TempHandle != NULL);
+            Status = EFI_INVALID_PARAMETER;

```

Comment:
What's the reason for this being EFI_INVALID_PARAMETER vs EFI_OUT_OF_RESOURCES used on line 1242?

Reply:
Should also be EFI_OUT_OF_RESOURCES - will update thanks :)